cli/tokens.go

```diff
@@ -58,16 +59,27 @@ func (r *RootCmd) createToken() *serpent.Command {
 			r.InitClient(client),
 		),
 		Handler: func(inv *serpent.Invocation) error {
-			res, err := client.CreateToken(inv.Context(), codersdk.Me, codersdk.CreateTokenRequest{
-				Lifetime:  tokenLifetime,
-				TokenName: name,
-			})
-			if err != nil {
-				return xerrors.Errorf("create tokens: %w", err)
+			if user != "" {
+				adminID := codersdk.Me
+				res, err := client.CreateTokenForUser(inv.Context(), adminID, user, codersdk.CreateTokenRequest{
+					Lifetime:  tokenLifetime,
+					TokenName: name,
+				})
+				if err != nil {
+					return xerrors.Errorf("create tokens for user %s: %w", user, err)
+				}
+				_, _ = fmt.Fprintln(inv.Stdout, res.Key) // Print the token to stdout
+			} else {
+				// Otherwise, create a token for the current user
+				res, err := client.CreateToken(inv.Context(), codersdk.Me, codersdk.CreateTokenRequest{
+					Lifetime:  tokenLifetime,
```

Comment:
I think you can actually keep this the same as it was before, but add a `userID` flag.

e.g.
```go
userID := codersdk.Me
if user != "" {
  userID = user
}
// Use the `userID` instead of `codersdk.Me` in `client.CreateToken`.
```